apps/server/src/shared/domain/entity/boardnode/board-do.builder.spec.ts

```diff
@@ -0,0 +1,173 @@
+import {
+	cardNodeFactory,
+	columnBoardNodeFactory,
+	columnNodeFactory,
+	setupEntities,
+	textElementNodeFactory,
+} from '@shared/testing';
+import { BoardDoBuilder } from './board-do.builder';
+import { BoardNodeType } from './types';
+
+describe(BoardDoBuilder.name, () => {
+	beforeAll(async () => {
+		await setupEntities();
+	});
+
+	describe('when building a column board', () => {
+		it('should work without descendants', () => {
```

Comment:
Our test structure entforce that every data setup and mocking must be part of the setup function. The setup must be part of the senario. It exists a couple of places where this must be changed in the PR. 
It is possible to move some more generall generations to top describe level. But each setup should be have his own setup with the parts that is defined in the sernaio description

Reply:
setupEntities is the function that initializes the orm entities when there is no database. Its not part of the szenario setup.
I went over all the tests, all setup is done either in a setup method as described, or specific to the testcase itself.